model/Licensing/Classes/CopyrightText.md

```diff
@@ -0,0 +1,27 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CopyrightText
```

Comment:
I strongly disagree, either that this is a shortcoming of SPDX 2.x or that this proposal is a simple change.

A copyright holder is not associated directly with a particular license. Rather, a copyright holder is associated with particular content for which they hold the copyright. In an SPDX document, this might be a Package as a whole, or one or more Files within it, or one or more Snippets within those Files. The copyright notice text is a property of the Package / File / Snippet, as is the corresponding license or licenses.

If it is necessary for an SPDX document creator to be more granular about which copyright notice text applies to a particular subunit of code, the appropriate way to handle that would be to model the Artifact at a lower level of granularity. For instance, if you need to talk about which license + copyright text is associated with one part of a File versus another, the right way to handle that would be to create Snippets corresponding to the different parts of the Files with differing license + copyright texts.

I think the proposed change is problematic because it introduces complexity which in my view is not necessary, and which imposes burdens on anyone who is implementing, consuming or creating licensing-related SPDX documents. The current modeling of copyrightText as a simple string which can be associated with any type of Artifact provides sufficient flexibility while being simple to use, consume and create.